patches/ungoogled-chromium/windows/windows-fix-building-without-safebrowsing.patch

```diff
@@ -3367,26 +3370,26 @@
  #include "chrome/browser/themes/theme_properties.h"
  #include "chrome/browser/ui/tab_modal_confirm_dialog.h"
  #include "chrome/browser/ui/views/chrome_typography.h"
-@@ -659,12 +654,6 @@ void DownloadItemView::ButtonPressed(vie
-   if (IsShowingDeepScanning() && sender == open_button_) {
-     content::WebContents* current_web_contents =
-         shelf_->browser()->tab_strip_model()->GetActiveWebContents();
--    open_now_modal_dialog_ = TabModalConfirmDialog::Create(
--        std::make_unique<safe_browsing::DeepScanningModalDialog>(
--            current_web_contents,
--            base::BindOnce(&DownloadItemView::OpenDownloadDuringAsyncScanning,
--                           weak_ptr_factory_.GetWeakPtr())),
--        current_web_contents);
-     return;
-   }
- 
-@@ -1062,15 +1051,10 @@ void DownloadItemView::ShowWarningDialog
+@@ -727,12 +722,6 @@ void DownloadItemView::ButtonPressed(vie
+     if (IsShowingDeepScanning()) {
+       content::WebContents* current_web_contents =
+           shelf_->browser()->tab_strip_model()->GetActiveWebContents();
+-      open_now_modal_dialog_ = TabModalConfirmDialog::Create(
+-          std::make_unique<safe_browsing::DeepScanningModalDialog>(
+-              current_web_contents,
+-              base::BindOnce(&DownloadItemView::OpenDownloadDuringAsyncScanning,
+-                             weak_ptr_factory_.GetWeakPtr())),
+-          current_web_contents);
```

Comment:
I haven't looked at this part of the code yet, but could this prevent an important callback from being invoked? I'd like to figure out why the CRX installation dialog doesn't show up at all (for #23), and I recall that drag-and-drop broke because we accidentally removed the invocation of the callback function.